site/hxapi/asynchronous-bookings.md

```diff
@@ -0,0 +1,143 @@
+---
+
+---
+
+# Asynchronous Bookings
+
+## Background
+
+We now offer products that use an asynchronous booking process. This allows partners to handle bookings for a wider range of products. However, asynchronous bookings require a different approach compared to synchronous bookings to provide a seamless experience for customers.
+This guide outlines:  
+- Best practices for integrating with our API for asynchronous bookings.  
+- Guidance on when and how to poll our API.  
+- Details on booking statuses.  
+- Information on enabling availability for async products.  
+
+---
+
+## Enabling Async Product Availability  
+
+To view all supply, including async products, partners can now specify whether they want these products included in the availability response.
+
+### Availability Request  
+
+Partners must include a new parameter in the availability request to indicate whether to include async products.  
+
+| Name                    | Data Type | Format  | Mandatory? | Additional Information                                    |
+|-------------------------|-----------|---------|------------|----------------------------------------------------------|
+| includeAsyncProducts    | Boolean   | `true`/`false` | N | If `true`, availability includes async products. Default is `false`. |
+
+### UK Products Availability Request
+
+{% codetabs %}
+{% codetab XML %}
+```
+https://api.holidayextras.co.uk/carpark/LGW?ABTANumber=YourABTANumber&Password=YourPassword&Initials=YourInitials&key=YourKey&token=YourToken&ArrivalDate=2020-01-07&ArrivalTime=1200&DepartDate=2020-01-08&DepartTime=1200&NumberOfPax=1&includeAsyncProducts=true
```

Comment:
```suggestion
https://api.holidayextras.co.uk/v1/carpark/LGW?ABTANumber=YourABTANumber&Password=YourPassword&Initials=YourInitials&key=YourKey&token=YourToken&ArrivalDate=2020-01-07&ArrivalTime=1200&DepartDate=2020-01-08&DepartTime=1200&NumberOfPax=1&includeAsyncProducts=true
```

Reply:
As I got the URL from https://docs.holidayextras.co.uk/hxapi/parking/av/carpark/ 
Should I also change the rest of the availability URLs?

Reply:
yes, we should always reference the versioned pathing, as otherwise we're adding to the mess that we're already in